packages/python-androguard/PKGBUILD

```diff
@@ -0,0 +1,39 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-androguard
+pkgver=3.2.1
+pkgrel=1
+pkgdesc='Androguard is a full python tool to play with Android files.'
+arch=('any')
+url='https://pypi.org/project/androguard'
+license=('MIT')
+depends=('python' 
+         'iptyhon')
+makedepends=('python-setuptools')
+optdepends=('python-pyperclip'
```

Comment:
Add some info, what the packages are used for
https://wiki.archlinux.org/index.php/PKGBUILD#optdepends

Reply:
I will